### DISCUSSION TITLE
issue while installing foundry on ubuntu

### DISCUSSION THREAD

**Discussion by Not-Sarthak** [Cyfrin/foundry-full-course-cu#1220] [Q&A] [answered]
![Screenshot from 2023-12-21 14-54-09](https://github.com/Cyfrin/foundry-full-course-f23/assets/92942966/bcc8db65-0943-40ad-9f24-bb3e79442c80)
while running the command `curl -L https://foundry.paradigm.xyz | bash` , it does not install foundry but instead the installation goes on and on without failing or succeeding

**Comment by Swarnim-Chandve** [comment]
I am facing the same problem, I even tried the methods in the older discussion posts but it still shows this error
I used the same command 
```curl -L https://foundry.paradigm.xyz | bash```
![Screenshot 2023-12-21 233440](https://github.com/Cyfrin/foundry-full-course-f23/assets/112816781/11062044-e11c-49c7-9694-e0a0d80c94a7)

**Comment by ghost** [comment, marked as answer]
@Not-Sarthak @Swarnim-Chandve 

Try the solution in [discussion 808](https://github.com/Cyfrin/foundry-full-course-f23/discussions/808).

**Reply by Not-Sarthak** [reply]
> @Not-Sarthak @Swarnim-Chandve 
> 
> Try the solution in [discussion 808](https://github.com/Cyfrin/foundry-full-course-f23/discussions/808).

okay, thank you!
